source/develop/release-management/features/virt/high-performance-vm.html.md

```diff
@@ -0,0 +1,176 @@
+
+## Usage
+
+ -  Setting a VM to run as a High Performance VM can be done to a new or existed VM via UI by selecting the "High Performance" type in the "Optimized for" pull down menu field displayed in VM new/edit dialog. 
+This new VM type is added in addition to already existed VM types: "Server", "Desktop". 
+ 
+ - In case of changing the VM type for a running VM, part of the required configuration changes will require a VM restart after saving. In case of changing the VM type for a new/existed VM, part of the required/suggested configuration may require manually changes of the VM's cluster configuration prior to setting the VM as High-performance, and part of the required/suggested configuration may require changing of the specific pinned host configuration prior to setting the VM as High-performance.
+
+ -  A High Performance Template or Pool can be created in the same way as a VM. In case the user wants to create or edit a VM to become High Performance, he can also choose a Template or a Pool which are configured as High Performance type and "inherit" this property for that specific VM.
+
+ -  Note that if an Instance Type is chosen for a VM, it has no influence on the VM high performance type and configuration. Nevertheless, configuration changed by the Instance Type may override the VM High Performance configuration.
+
+ - Once the "High Performance" VM type was chosen in the VM/Template/Pool dialog,  a set of automatic configuration changes, and suggested manual configuration changes are proposed to the user in order to suggest the user with the optimal configuration setting:
+	 - A list of validations and suggested manual configuration changes are displayed in a pop-up window on save ,and the user can choose if to accept and perform before saving or ignore them.
+	 
+	 - A list of configuration changes will be applied automatically (and the user can cancel before saving via WebAdmin UI). We should consider displaying that list of changes in a pop-up or tool-tip or any other suggested solution.
+
+ - This new VM type is supported only for non Windows OS and therefore in case of choosing a Windows based OS in the VM/Template/Pool dialog, the High Performance VM type can not be chosen.
+
+ -  Once the user accept the configuration and click the OK button within the pop-up dialog, the configuration is changed accordingly and the VM becomes a High Performance VM.
+
+ - a new icon is displayed for this new High Performance VM type, left to the VM name, and the VM type will also be displayed in the "General" sub-tab (it is required since no icons are displayed for Pools and Templates).
+      
+ - This feature doesn't work on all cluster levels (since huge pages are not supported prior to oVirt 4.2).
+ 
+ - Note that all of the scenarios described above can be done in UI only via oVirt WebAdmin and not via the UserPortal.
```

Comment:
we intend to replace it in 4.2 so the "old" one is no longer relevant.

I've meant to differentiate the display by e.g. an icon. But indeed some of the configuration cannot be done. That's fine I guess, as displaying such VMs (and ability to run/stop) should be enough